### ISSUE TITLE
fix: keyboard focus does not return to previous element after closing FluentDialog

### ISSUE THREAD

**Issue by adamint** [microsoft/fluentui-blazor#4094]
# 🐛 Bug Report

As in title, keyboard focus should be placed back on the element focused before the dialog was opened after it has been closed.

## 💻 Repro or Code Sample

Can be reproduced in any of the FluentDialog examples -> https://www.fluentui-blazor.net/Dialog

## 💁 Possible Solution

Register an onkeydown listener on both the first menu item (if one exists) and the menu button. If focus is on the menu button, tab will place focus on the first item in the menu. If focus is on the first menu item, a shift-tab will place focus back on the button.

Save a reference to the element currently focused during dialog open, in the dialog service. Focus it again during close.

I will follow up with a fix tomorrow.

## 🔦 Context

https://github.com/dotnet/aspire/issues/10119

**Comment by vnbaaij** [comment]
Fixed with PR #4095
